test/common.js

```diff
@@ -18,10 +19,38 @@ if (process.env.TEST_THREAD_ID) {
 }
 exports.tmpDir = path.join(exports.testDir, exports.tmpDirName);
 
-exports.opensslCli = path.join(path.dirname(process.execPath), 'openssl-cli');
+var opensslCli = null;
+
+
+function opensslCliGetter() {
```

Comment:
@bnoordhuis Splitting is just for easy to read and notify this is a getter function. I don't mind to change it into inline.